docs/mkdocs/en/model.md

```
@@ -2291,6 +2291,36 @@ model := anthropic.New("claude-3-5-sonnet",
 
 For detailed explanations of the token calculation formula, tailoring strategy, and custom strategy implementation, please refer to [Token Tailoring under OpenAI Model](#3-token-tailoring).
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)